app/javascript/components/mentoring/session/Guidance.tsx

```diff
@@ -109,12 +112,18 @@ export const Guidance = ({
                 Try and guide the student towards this solution. It is the best
                 place for them to reach at this point during the Track.
               </p>
-              <pre className="overflow-auto">
-                <code
-                  className={language}
-                  dangerouslySetInnerHTML={{ __html: exemplarSolution }}
-                />
-              </pre>
+              {exemplarFiles.map((file) => {
+                return (
+                  <div key={file.filename} className="exemplar-files">
+                    {exemplarFiles.length > 1 ? (
```

Comment:
@kntsoriano I added this so that the filename only shows if there are multiple. That's broken a couple of tests. Could you get them green pls.